src/app/chat-list/chat-list.component.html

```diff
@@ -1,9 +1,19 @@
 <mat-action-list>
   <button
     mat-list-item
-    *ngFor="let chatId of chats"
-    (click)="onChatSelect(chatId)"
+    (click)="onChatSelect(chat.id)"
+    *ngFor="let chat of chats"
+    class="h-auto p-0 py-2"
   >
-    {{ chatId }}
+    <div class="container">
+      <div class="row">
+        <div class="col-auto ps-0">
+          <img class="rounded-circle" height="56px" [src]="chat.imageUrl" />
+        </div>
+        <div class="col">
+          <h3 class="m-0">{{ chat.title }}</h3>
```

Comment:
chat.title should be changed to chat.groupName to work with create group component